src/components/Modal/node/SendMessageModal.tsx

```diff
@@ -67,6 +75,28 @@ type SendMessageModalProps = {
   tooltip?: JSX.Element | string;
 };
 
+function sortAddresses(
+  peers: GetPeersResponseType | null,
+  me: AddressesType,
+  peerIdToAliasLink: {
+    [peerId: string]: string
+  }
+) : string[] {
+  if(!peers || !me) return [];
+  const connectedPeers = peers.connected;
+  const myAddress = me.hopr as string;
+  const peerIdsWithAliases = Object.keys(peerIdToAliasLink);
+  if(peerIdsWithAliases.length === 0) return [myAddress, ...connectedPeers.map(peer => peer.peerId)];
```

Comment:
you are not checking if connected peers contain `myAddress`.
This is something that you would do if you have peerIdsWithAliases (see line 95)

Reply:
Re-did whole thing